### PR TITLE
Add descriptions

### PR DESCRIPTION
**Pull request by ozdanborne**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit d58253811332a65d9e09f058babd0f1b0ce5c55c

https://deploy-preview-3044--calico.netlify.com

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 4c9a1fa296fdf38195bd9dda5434330870a13bad

https://deploy-preview-3044--calico.netlify.com

File: master/networking/workloads-outside-cluster.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Configure workload access outside the cluster
+description: Describes how to enable and disable internet access for your workloads.
```
Comment (caseydavenport) [inline master/networking/workloads-outside-cluster.md]: I suggest ditching the "Describes how to" preamble on those which have it. WDYT?

More to the point "How to enable and disable internet access for your workloads"

Comment (bmckercher123) [reply master/networking/workloads-outside-cluster.md]: Agree.

File: master/networking/openstack/semantics.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Detailed semantics
+description: Describes connectivity that {{site.prodname}} provides, how and why this connectivity is different from traditional Neutron API semantics.
```
Comment (lmm) [inline master/networking/openstack/semantics.md]: I didn't think this worked

File: master/networking/enabling-ipvs.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
-title: Enabling IPVS in Kubernetes
+title: Enable IPVS in Kubernetes
+description: Use IPVS kue-proxy for load balancing traffic across pods
```
Comment (lmm) [inline master/networking/enabling-ipvs.md]: `kube-proxy`

File: master/networking/openstack/semantics.md (hunk 2 above)
Comment (bmckercher123) [reply master/networking/openstack/semantics.md]: Last I checked with Neil a few months ago, he said he believed the OpenStack content was accurate and up to date. I took this description directly from the doc. If it is not, correct maybe we should open separate PR to review the doc?

Comment (bmckercher123) [reply master/networking/openstack/semantics.md]: Fixed

**Pull request merged**